### PR TITLE
Edits to tutorial

### PR DESCRIPTION
**Pull request by lindsayplatt**
This follows on to the incredible work @steeleb made in #37! I went through and read the tutorial from start to end and adjusted/added content as I came across places that may cause confusion. Once this is merged, I am confident we can close all the issues mentioned - #10, #14, #17, #13. I think we can open new issues for things that arise from your class on Monday.

There is one section that I just inserted `?????` into for now and I'm looking for your help. What do we want people to do to close out the "validation" part? I inserted a spot for us to direct those doing the validation script to complete it in `Finishing up the labeling` but didn't know what that looked like. I remember you made a script for it that I reviewed ...

### CODE REVIEW

File: docs/index.Rmd [hunk 1]
```diff
@@ -269,19 +273,23 @@ When you have completed labeling your image, it's time to export the labels you
 
   
 
-That's all! If you're ready to label another image, repeat from 'How to label: Part 1'. See the section 'Selecting a mission-date for the `eeplumB.js` script'. You do not need to wait for the task to complete to navigate away from this screen or to start a new script.
+That's all! You do not need to wait for the task to complete to navigate away from this screen or to start a new script.
+
+- If you were completing the validation workflow, check your labeled images against our expectations by ????. If you are happy with your results, you are ready to start labeling for real! Head to the `Selecting a mission-date for labeling` section to pick your own mission-date, and then start back at `How to label: Part 1`.
```
Comment (steeleb) [inline docs/index.Rmd:278]: https://code.earthengine.google.com/7dd1bca1b1a8f80dd51e045487db9a5a?hideCode=true
- If you were completing the validation workflow, check your labeled images against our expectations by heading to [this labeled validation set](https://code.earthengine.google.com/7dd1bca1b1a8f80dd51e045487db9a5a?hideCode=true). Click 'Run' and click through the Tile-Dates, comparing your labels to ours. Did you use the label classes similarly? Remember, it's less about how many pixels you've labeled and more that the 'quality' of the pixel label is high.

File: docs/index.Rmd [hunk 2]
```diff
@@ -269,19 +273,23 @@ When you have completed labeling your image, it's time to export the labels you
...
+To help us stay organized and avoid duplication, we have listed all our target mission-date pairs in [this Google Sheet](https://docs.google.com/spreadsheets/d/1g5owPatzZOKuidQccaa5fUGVLnUtHD2na3zjBfQhB-g/edit?usp=sharing). These mission-date pairs have been sorted randomly. As a general rule of thumb, know that Landsat 5 ('mission' = LS5 in the Google sheet) images are 'more difficult' to label because the colors are generally more murky than any other mission. We suggest starting with a LS8 or LS9 image if you are new to satellite imagery.
 
 When you've selected a mission-date to label, put your name and initials in the proper row **when you start** to label the image. This will prevent unnecessary duplication of labeling. When you're finished labeling and have exported the data, add 'yes' to the `finished?` column.
 
-The row values for the 'mission' and 'date' column in the Google Sheet:
+To start labeling your selected mission-date, head to the `How to label: Part 1` section. Note that in steps 6 & 8, you will need to use the row values for the 'mission' and 'date' column from the Google Sheet
```
Comment (steeleb) [inline docs/index.Rmd:288]: To start labeling your selected mission-date, head to the `How to label: Part 1` section. Note that in steps 6 & 8, you will need to use the row values for the 'mission' and 'date' column from the Google Sheet in addition to editing the script to include your initials on lines 14, 16, and 18 after you've imported the geometry records.

**Comment by steeleb** [comment]
You're a rockstar.

This is absolutely excellent:
Screenshot 2023-04-17 at 3 12 45 PM

I really appreciate all the clarity you added. 

No notable feedback from the class - I gave them an intro to crowd-sourced science and image segmentation and had them do the validation script. I hope a few of them want to continue to label. We'll see! I'm going to follow up with a PDF of my slides and a link to the live site hoping that might get some folks to procrastinate through labeling. :)

**Review by steeleb — CHANGES_REQUESTED** [review]
Just two details and then I'll merge this sucker.

**Comment by steeleb** [comment]
Also, I'm going to attach those issues to this PR.

File: docs/index.Rmd (hunk 1 above)
Comment (lindsayplatt) [reply docs/index.Rmd:278]:
```suggestion
- If you were completing the validation workflow, check your labeled images against our expectations by heading to [this labeled validation set](https://code.earthengine.google.com/7dd1bca1b1a8f80dd51e045487db9a5a?hideCode=true).  Click 'Run' and click through the Tile-Dates, comparing your labels to ours. Did you use the label classes similarly? Remember, it's less about how many pixels you've labeled and more that the 'quality' of the pixel label is high. If you are happy with your results, you are ready to start labeling for real! Head to the `Selecting a mission-date for labeling` section to pick your own mission-date, and then start back at `How to label: Part 1`.
```

File: docs/index.Rmd (hunk 2 above)
Comment (lindsayplatt) [reply docs/index.Rmd:288]:
```suggestion
To start labeling your selected mission-date, head to the `How to label: Part 1` section. Note that in steps 6 & 8, you will need to use the row values for the 'mission' and 'date' column from the Google Sheet in addition to editing the script to include your initials (lines 14, 16, and 18) after you've imported the geometry records.
```

**Comment by steeleb** [comment]
Wahooooo!!

**Review by steeleb — APPROVED** [review]

**Pull request merged**